MOBILE_CLIENT/COMMON/SECURE/SECURE_REPUTATION/description.md

```diff
@@ -0,0 +1 @@
+Reputation analysis for domain names and IP addresses scanned by multiple APIs and public databases.
```

Comment:
If this is about the reputation of the asset being good we should mention it

Reply:
Thats not always the case, the asset could be unreported.

Reply:
in that case `risk_rating` should be info not secure